src/main/java/spireMapOverhaul/zones/windy/events/WindyBridgeEvent.java

```diff
@@ -0,0 +1,294 @@
+                        newBodyText += desc.get("typeA");
+                        break;
+                    case SKILL:
+                        newBodyText += desc.get("typeS");
+                        break;
+                    case POWER:
+                        newBodyText += desc.get("typeP");
+                        break;
+                    default:
+                        newBodyText += desc.get("typeU");
+                }
+                break;
+        }
+
+        if(AbstractDungeon.miscRng.randomBoolean(getLossProbability() * 0.01f)){
+            CardCrawlGame.sound.play("CARD_EXHAUST");
+            lostItem = true;
+            newBodyText += desc.get("loseItem");
+            switch (c){
+                case CARD:
+                    AbstractDungeon.effectList.add(new PurgeCardEffect(couldLoseCard));
+                    AbstractDungeon.player.masterDeck.removeCard(couldLoseCard);
+                    newBodyText += FontHelper.colorString(couldLoseCard.name, "r");
+                    break;
+                case GOLD:
+                    newBodyText += FontHelper.colorString("" + Math.min(GOLD_LOSS, AbstractDungeon.player.gold), "r") + desc.get("gold");
+                    AbstractDungeon.player.loseGold(GOLD_LOSS);
+                    break;
+                case POTION:
+                    AbstractDungeon.player.removePotion(couldLosePotion);
+                    newBodyText += FontHelper.colorString(couldLosePotion.name, "r");
+                    break;
+                case RELIC:
+                    AbstractDungeon.player.loseRelic(couldLoseRelic.relicId);
+                    newBodyText += FontHelper.colorString(couldLoseRelic.name, "r");
+                    break;
+                default:
+                    SpireAnniversary6Mod.logger.info("Windy zone event broken!??!?! Spam the steam comment section about this bug and call GK mean names.");
+            }
+            newBodyText += desc.get("loseItem2");
+        }
+
+        if(optionNum >= 0){
+            optionCons.remove(optionNum);
+            optionPros.remove(optionNum);
+        }
+        imageEventText.updateBodyText(newBodyText);
+    }
+
+    public static int getLossProbability(){
+        if(AbstractDungeon.ascensionLevel >= 15){
+            return 40;
+        }
+        return 25;
+    }
+
+    // method sampled from Nloth event
+    public AbstractRelic getLosableRelic(){
+        ArrayList<AbstractRelic> relics = new ArrayList<>(AbstractDungeon.player.relics);
+        Collections.shuffle(relics, new Random(AbstractDungeon.miscRng.randomLong()));
+        return relics.size() > 0 ? relics.get(0) : null;
+    }
+
+    //identify a random highest tier card that can be lost
+    public AbstractCard getLosableCard(){
+        CardGroup group = CardGroup.getGroupWithoutBottledCards(AbstractDungeon.player.masterDeck);
+        couldLoseCard = group.getRandomCard(AbstractDungeon.eventRng, AbstractCard.CardRarity.RARE);
```

Comment:
Man, events are annoying with the rngs they use. Some use cardRng for card stuff like gremlin match while others use miscRng for basically everything.

Since you went the misc rng route, maybe keep that that way here? eventRng should probably only be used for spawning events

Reply:
Whoops, missed these ones. I'll set them to miscRng.